### DISCUSSION TITLE
[2023-07-31] GUI Team Demo day

### DISCUSSION THREAD

**Discussion by farmaazon** [enso-org/enso#7443] [Weekly Demo Day]
* [New panel arrangement is merged](#7350). There were many conflicts, and some resolutions brought errors, and I need to spend some time on fixing them.
* [Fixed panics when closing projects in certain situations](#7422) 
* [Fixed missing and not working preview in Component Browser](#7441). Discovered them during [investigating another issue](#7357) and I decided to fix it. PR is ready for review.

**Comment by Frizi** [comment]
- finished and merged new node design #7311 
- working on gracefuly handling WebAssembly panics #7376

**Comment by MichaelMauderer** [comment]
Finished
* Refactor CB Breadcrumbs into a self contained crate #7363
* Synchronize CB Breadcrumbs with documentation panel #7310 (awaiting rebase)
* Rendering issue with Text visualisation #7332

**Comment by kazcw** [comment]
- Fixed a bug parsing invalid syntaxes related to a lexical special case. This was discovered because of #7357.
- CB focus: Fixed a bug in the latest design updates, and merged.
- New search: Fixed a logical conflict with a recent CB change and merged.
- Debugging a strange issue, haven't determined yet if it's related to #7284 or a hardware problem on my machine.

**Comment by mwu-tow** [comment]
Most of the work was spent on #7373. There is some delay, I expect to have PR ready tomorrow.
The old statusbar and popup code paths were replaced with the new notification system.
Some basic styling was done to make it roughly follow the overall theme.
![obraz](https://github.com/enso-org/enso/assets/1548407/5d15f03e-c126-4251-aa63-26ab88c8e96c)
![obraz](https://github.com/enso-org/enso/assets/1548407/db7ecd81-4deb-4b03-85f3-dcd23cf2c03f)

I did some CI work (#7407):
* fixing engine work cancelling;
* fixing cloud releases.

**Comment by JaroslavTulach** [comment]
Engine team is finishing work on:
- #7101
